.github/dependabot.yml

```
@@ -1,7 +1,12 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Is daily really needed?

```suggestion
      interval: weekly
```

---

Yeah let's not do daily. It spams so much 🙃

---

I think it's nice to not have it on the same day as the pip bumps.

---

And the action bumps for the actions we normally use don't spam very much in my experience.